CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
+- Fixed automatic opening the Reverb settings page when there were no active midi devices https://github.com/GrandOrgue/grandorgue/issues/1002
 # 3.6.1 (2022-01-29)
-- Updating French, German, Polish, Spanish and Swedish translations of the Settings Dialog https://github.com/GrandOrgue/grandorgue/discussions/934
+- Updated French, German, Polish, Spanish and Swedish translations of the Settings Dialog https://github.com/GrandOrgue/grandorgue/discussions/934
```

Comment:
Usually, changing a passed version is strange even if there is a mistake.

Reply:
> Usually, changing a passed version is strange even if there is a mistake.

Yes, agree, but later is better than never.